cosmo_tester/framework/test_hosts.py

```diff
@@ -371,7 +371,10 @@ def _create_config_file(self):
         return config_file
 
     def bootstrap(self):
-        manager_install_rpm = util.get_manager_install_rpm_url()
+        specific_manager_install_rpm = ATTRIBUTES.cloudify_manager_install_rpm_url.strip()
+        manager_install_rpm = specific_manager_install_rpm \
+            if specific_manager_install_rpm else util.get_manager_install_rpm_url()
```

Comment:
This might be better written as:
manager_install_rpm = ATTRIBUTES.cloudify_manager_install_rpm_url.strip() or util.get_manager_install_rpm_url()